M2/Macaulay2/editors/prism/package.json

```diff
@@ -9,6 +9,6 @@
     "css-loader": "^7.1.2",
     "style-loader": "^4.0.0",
     "webpack": "^5.96.1",
-    "webpack-cli": "^5.0.1"
+    "webpack-cli": "^6.0.1"
```

Comment:
Is there a way to just put "latest" here once and for all?

Reply:
I think that's probably not a great idea in general because there might be breaking changes.  In this case, there weren't any -- I re-compiled prism.js with the new version of webpack, it worked, and in fact the resulting file was identical to the one produced by the old webpack.  But there might be.  For example, he Visualize package is full of code that breaks if we upgrade its npm dependencies.